swatch-contracts/src/main/resources/application.properties

```
@@ -1,4 +1,5 @@
 SERVER_PORT=${clowder.endpoints.swatch-contracts.port:8000}
+SWATCH_INTERNAL_SUBSCRIPTION_ENDPOINT=http://${clowder.endpoints.capacity-ingress.hostname}:${clowder.endpoints.capacity-ingress.port}
```

If we instead set this to:

```suggestion
SWATCH_INTERNAL_SUBSCRIPTION_ENDPOINT=http://swatch-subscription-sync-service:8000/api/rhsm-subscriptions/v1
```

Then we don't have to override via template var/env var anywhere :smile: